fossunited/ticketing/doctype/foss_event_ticket_transfer/foss_event_ticket_transfer.py

```diff
@@ -53,9 +53,9 @@ def validate_status_is_pending(self):
             )
 
     def transfer_ticket(self):
-        self.validate_ticket_exists()
```

Comment:
not sure why we dont need to validate that the ticket exists anymore. shouldn't we still be doing it? this was preventing a valid issue that we had earlier.

Reply:
It was redundant with line 37 in the same file